src/changelog-check.ts

```diff
@@ -229,6 +256,23 @@ async function getChangedPackages(
 
     const packageInfo = getPackageInfo(file, workspacePatterns);
     if (packageInfo) {
+      // Check if we've already determined if this package is private
+      let isPrivate = privatePackageCache.get(packageInfo.package);
+      if (isPrivate === undefined) {
+        // If not in cache, check and cache the result
```

Comment:
nit: These comments are extraneous.
```suggestion
      let isPrivate = privatePackageCache.get(packageInfo.package);
      if (isPrivate === undefined) {
```